transaction/transaction.go

```diff
@@ -84,3 +155,15 @@ func estimateSize(txn types.Transaction) (uint64, error) {
 	}
 	return uint64(len(stx)), nil
 }
+
+func byteArrayFromBase64(in string) (out [32]byte, err error) {
```

Comment:
Consider removing this wrapper and doing it inline to match the rest of the code. If the point is to make sure it's 32 bytes, I'd change the name of the function. Also, I think it's worth using a constant instead of directly 32.

Reply:
I would like to preserve the length check. Is this better?